website/public/js/controllers/partyCtrl.js

```diff
@@ -51,6 +51,11 @@ habitrpg.controller("PartyCtrl", ['$rootScope','$scope','Groups','Chat','User','
           Analytics.track({'hitType':'event','eventCategory':'button','eventAction':'click','eventLabel':'Leave Party'});
           $scope.selectedGroup = group;
           $scope.popoverEl = $($event.target);
+
+          if ($scope.popoverEl[0].localName == "span") {
+              $scope.popoverEl = $($scope.popoverEl[0].parentNode);
+          }
```

Comment:
You could just do
```
$scope.popoverEl = $($event.target).closest('.btn');
```
`closest` starts at the element and searches up the tree till it finds a match.  In this case it will just be the el itself or it's parent. I tried it out and it seems to work.